client/webserver/site/src/js/forms.js

```diff
@@ -1,6 +1,15 @@
 import Doc from './doc'
 import { postJSON } from './http'
 
+var configInputField = (option) => {
+  return `
+  <div>
+    <label for="${option.key}" class="pl-1 mb-1 small">${option.displayname || option.key}</label>
+    <input type="text" class="form-control select" placeholder="${option.description}" id="${option.key}">
```

Comment:
Use a `data-` attribute instead of ID here. We don't always have control over what the keys are, and `id` must be unique.